var/spack/repos/builtin/packages/py-proj/package.py

```diff
@@ -0,0 +1,28 @@
+from spack import *
+
+class PyProj(Package):
+    """Python interface to the PROJ.4 Library."""
+    homepage = "http://jswhit.github.io/pyproj/"
+    url      = "https://github.com/jswhit/pyproj/tarball/v1.9.5.1rel"
+
+
+    version('1.9.5.1', 'a4b80d7170fc82aee363d7f980279835')
+
+    # We need the benefits of this PR
+    # https://github.com/jswhit/pyproj/pull/54
+    version('citibeth-latlong2',
+        git='https://github.com/citibeth/pyproj.git',
+        branch='latlong2')
+
+    extends('python')
+
+    depends_on('py-cython')
+    depends_on('py-setuptools')
```

Comment:
I agree.  But a grep through the repo reveals some confusion over
type='build' vs. type=nolink. My guess is type=`build` is right, and
type=nolink needs to be converted.  Your opinions?

I'll fix the packages below that are mine (glint2, ibmisc, icebin, py-proj).

```
cantera/package.py:    depends_on('py-cython', when='+python', type=nolink)
glint2/package.py:    depends_on('py-cython', when='+python')
ibmisc/package.py:    depends_on('py-cython', when='+python', type=nolink)
icebin/package.py:    depends_on('py-cython', when='+python')
py-h5py/package.py:    depends_on('py-cython@0.19:', type='build')
py-iminuit/package.py:    depends_on('py-cython', type='build')
py-netcdf/package.py:    depends_on('py-cython', type=nolink)
py-proj/package.py:    depends_on('py-cython')
py-pytables/package.py:    depends_on('py-cython', type=nolink)
```


On Fri, Sep 2, 2016 at 9:26 AM, Adam J. Stewart <notifications@github.com>
wrote:

> In var/spack/repos/builtin/packages/py-proj/package.py:
>
> > +    homepage = "http://jswhit.github.io/pyproj/"
> > +    url      = "https://github.com/jswhit/pyproj/tarball/v1.9.5.1rel"
> > +
> > +
> > +    version('1.9.5.1', 'a4b80d7170fc82aee363d7f980279835')
> > +
> > +    # We need the benefits of this PR
> > +    # https://github.com/jswhit/pyproj/pull/54
> > +    version('citibeth-latlong2',
> > +        git='https://github.com/citibeth/pyproj.git',
> > +        branch='latlong2')
> > +
> > +    extends('python')
> > +
> > +    depends_on('py-cython')
> > +    depends_on('py-setuptools')
>
> py-setuptools is probably a build dependency? Not sure about py-cython.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/LLNL/spack/pull/717/files/552a2815cd82b04c74eac44352f5e812ea0fa7f6#r77345291>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB1cd69ohdDBe-Im4zrFwehp40mIpyubks5qmCQOgaJpZM4H9_nz>
> .
>